### PR TITLE
vpa and card in payout bene api

### PR DESCRIPTION
**Pull request by shubhamatlani**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.cashfree</groupId>
 	<artifactId>cashfreelibjava</artifactId>
-	<version>0.0.5-SNAPSHOT</version>
+	<version>0.0.5</version>
```
Comment (ankur-cashfree) [inline pom.xml:6]: will need to increment this for next release

File: src/main/java/com/cashfree/lib/payout/domains/response/GetBeneficiaryResponse.java [hunk 2]
```diff
@@ -35,6 +33,8 @@
 
     private String ifsc;
 
+    private String vpa;
+
     private String address1;
```
Comment (ankur-cashfree) [inline src/main/java/com/cashfree/lib/payout/domains/response/GetBeneficiaryResponse.java:38]: will also need to add maskedCard in the response POJO

File: pom.xml (hunk 1 above)
Comment (shubhamatlani) [reply pom.xml:6]: 0.0.5 is the expected next version that is yet to be released:
https://mvnrepository.com/artifact/com.cashfree/cashfreelibjava

**Review by ankur-cashfree — APPROVED** [review]
Looks good

**Pull request merged**